### ISSUE TITLE
Getting vulkan validation errors whilst running mangohud with a vulkan app

### ISSUE THREAD

**Issue by codedcosmos** [flightlessmango/MangoHud#728]
I'm working on a vulkan application, and just wanted to quickly check performance so I decided to use mangohud instead of implementing an fps counter...

Either way I have validation layers enabled, specifically `VK_LAYER_KHRONOS_validation`

I just wanted to write a bug report because when running mangohud I get these validation errors.
```
VUID-VkMappedMemoryRange-size-01389(ERROR / SPEC): msgNum: 249857837 - Validation Error: [ VUID-VkMappedMemoryRange-size-01389 ] Object 0: handle = 0x9a21e90000000ad3, type = VK_OBJECT_TYPE_DEVICE_MEMORY; | MessageID = 0xee4872d | vkFlushMappedMemoryRanges: Size in pMemRanges[1] is VK_WHOLE_SIZE and the mapping end (0xc6c = 0x0 + 0xc6c) not a multiple of VkPhysicalDeviceLimits::nonCoherentAtomSize (0x40) and not equal to the end of the memory object (0xd00). The Vulkan spec states: If size is equal to VK_WHOLE_SIZE, the end of the current mapping of memory must either be a multiple of VkPhysicalDeviceLimits::nonCoherentAtomSize bytes from the beginning of the memory object, or be equal to the end of the memory object (https://vulkan.lunarg.com/doc/view/1.3.204.1/linux/1.3-extensions/vkspec.html#VUID-VkMappedMemoryRange-size-01389)
    Objects: 1
        [0] 0x9a21e90000000ad3, type: 8, name: NULL
```

When I'm running it without mangohud I'm not getting any of these errors. I'm also not even calling `vkFlushMappedMemoryRanges` once and I'm not using `VK_WHOLE_SIZE` anywhere.

If this is actually my fault and I'm supposed to implement an extension to use mangohud or something I apologise. If that is the case please close this issue.

**Comment by jackun** [comment]
_Should_ be fixed [since v0.6.2](https://github.com/flightlessmango/MangoHud/commit/6c737dc7349dc891da7a280e069e5c5a09b2d1f9)

**Comment by codedcosmos** [comment]
Yeah, Ubuntu installed 6.1-1
I'm gonna close this as it's my fault.
